src/assets/scripts/client/InputController.js

```diff
@@ -638,68 +634,68 @@ export default class InputController {
     tab_completion_match() {
         let matches;
         const val = this.$commandInput.val();
-        let aircrafts = prop.aircraft.list;
+        let aircrafts = this._aircraftController.aircraft.list;
```

Comment:
yeah.  It bugs me, but I'm trying to limit the scope of these changes as much as possible